src/Combinatorics/Graphs/functions.jl

```diff
@@ -788,6 +850,47 @@ false
 """
 is_isomorphic(g1::Graph{T}, g2::Graph{T}) where {T <: Union{Directed, Undirected}} = Polymake.graph.isomorphic(pm_object(g1), pm_object(g2))::Bool
 
+@doc raw"""
+    is_isomorphic_with_permutation(G1::Graph, G2::Graph) -> Bool, Vector{Int}
+
+Return whether `G1` is isomorphic to `G2` as well as a permutation
+of the nodes of `G1` such that both graphs agree.
+
+# Examples
+```jldoctest
+julia> is_isomorphic_with_permutation(edgegraph(simplex(3)), dualgraph(simplex(3)))
+(true, [1, 2, 3, 4])
+
+```
+"""
+function is_isomorphic_with_permutation(G1::Graph, G2::Graph)
+  f12 = Polymake.graph.find_node_permutation(G1.pm_graph, G2.pm_graph)
+  if isnothing(f12)
+    return false, Vector{Int}()
+  end
+  return true, Polymake.to_one_based_indexing(f12)
+end
+
+@doc raw"""
+    _is_equal_up_to_permutation_with_permutation(A1::MatElem, A2::MatElem) -> Bool, Vector{Int}
+
+Return a permutation `I` such that `A1[I,I] == A2` and whether it exists.
+
+The method assumes that both matrices are symmetric, their diagonal entries
+are all equal (and so irrelevant) and the off-diagonal entries are either ``0``
+or ``1``.
```

Comment:
Just a nitpick:
You require matrices to be symmeric here, but further up in graph_from_adjacency_matrix, you only looked at the lower triangular part. This makes perfect sense to me, but it might be useful to state it more clearly (entries above the diagonal are ignored!).